.github/workflows/post-release.yml

```diff
@@ -73,7 +74,15 @@ jobs:
             -a tag=${{ env.LIFECYCLE_VERSION }} \
             buildpacksio/lifecycle:${{ env.LIFECYCLE_VERSION }}@${MANIFEST_SHA}
           cosign verify -key cosign.pub -a tag=${{ env.LIFECYCLE_VERSION }} buildpacksio/lifecycle:${{ env.LIFECYCLE_VERSION }}
+
           cosign attach sbom -sbom ./*-bom.cdx.json -type cyclonedx buildpacksio/lifecycle:${{ env.LIFECYCLE_VERSION }}
+          SBOM_SHA=$(cat *-checksums.txt | grep bom.cdx.json | awk '{print $1}')
```

Comment:
Are we using this somewhere? Can't immediately spot where it's being used.

Reply:
Added in 707398c

Reply:
Oh whoops... that sha is the sha of the layer, not the manifest!

Reply:
Removed the SBOM sha for now.